var/spack/repos/builtin/packages/flecsi/package.py

```diff
@@ -67,7 +67,10 @@ class Flecsi(CMakePackage, CudaPackage):
     # All Current FleCSI Releases
     for level in ('low', 'medium', 'high'):
         depends_on('caliper@2.0.1~adiak~libdw', when='@:1.9 caliper_detail=%s' % level)
-        depends_on('caliper@2.4.0~libdw', when='@2.0: caliper_detail=%s' % level)
+        depends_on('caliper', when='@2.0: caliper_detail=%s' % level)
+        conflicts('caliper@2.6', when='@2.0: caliper_detail=%s' % level)
+        conflicts('caliper@2.7', when='@2.0: caliper_detail=%s' % level)
```

Comment:
I wonder if this would be better:
```suggestion
        depends_on('caliper@:2.5', when='@2.0: caliper_detail=%s' % level)
```
since it also excludes `master`. Or is `master` working correctly with caliper?

Reply:
`caliper@master` is working correctly with flecsi if that was the question.